.github/workflows/auto_cherry_pick_merged.yaml

```diff
@@ -0,0 +1,164 @@
+### The workflow for retrying/rerunning the merged PRs AutoCherryPick which was missed/failed due to any circumstances
+name: Retry Merged PRs AutoCherryPick
+
+# Run on workflow dispatch from CI
+on:
+  workflow_dispatch:
+    inputs:
+      parentPR:
+        type: string
+        description: An identifier for parent PR to retry its cherrypick
+      branches:
+        type: string
+        description: Comma separated list of branches where the master PR to be cherrypicked
+
+env:
+  number: ${{ github.event.inputs.parentPR }}
+  is_dependabot_pr: ''
+
+jobs:
+  get-parentPR-details:
```

Comment:
```suggestion
  get-parentPR-detail's:
```

Reply:
Not changing as this would be used as Id later and hence we cant use such characters there.